python/python/glide/async_commands/core.py

```diff
@@ -1496,6 +1496,94 @@ async def linsert(
             ),
         )
 
+    async def lmove(
+        self,
+        source: str,
+        destination: str,
+        wherefrom: ListDirection,
+        whereto: ListDirection,
```

Comment:
in python the syntax is:
```suggestion
        where_from: ListDirection,
        where_to: ListDirection,
```